### DISCUSSION TITLE
[Feature] Filter search results by album

### DISCUSSION THREAD

**Discussion by tezlm** [immich-app/immich#8702] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be great if there was a search option to filter images by album. Right now, I can search for images that aren't in any album, but there's no way to search in a single (or multiple) album(s).

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2407
